index.html

```diff
@@ -0,0 +1,207 @@
+          <li><a class="campaign nav-link" href="">CAMPAIGN</a></li>
+        </ul>
+      </nav>
+    </header>
+    <section class="header">
+      <div class="background">
+        <nav>
+          <i class="fa fa-bars menu"></i>
+        </nav>
+        <span>"HELLO! AFRICAN WOMEN"</span>
+        <h1>
+          EMPOWERING AFRICAN WOMEN <br />
+          GLOBAL SUMMIT <br />
+          2023
+        </h1>
+        <p>
+          A joyful celebration believing in the value of empowering women to
+          achieve their destiny by removing barriers that hold them back with
+          people from over 54 countries is taking place in April, Ethiopia.
+        </p>
+        <div class="headerDate">2023.05.20(THU) ~ 21(FRI)</div>
+        <div class="headerAt text-secondary mb-5">
+          @African women empowerment
+        </div>
+      </div>
+    </section>
+
+    <section class="program">
+      <h2>Main Program</h2>
+      <hr />
+      <ul class="program-ul">
+        <li>
+          <div>
+            <img src="./images/lecture.svg" alt="lecture logo " />
+          </div>
+          <span>Lecture</span>
+          <p class="right">
+            Listen to the speakears from various countries about African women
+            empowerment.
+          </p>
+        </li>
+        <li>
+          <div>
+            <img src="./images/cc.svg" alt="exhibition logo " />
+          </div>
+          <span>Exhibition</span>
+          <p class="right">
+            Appreciate various creations applying AWE license of artists,
+            organized from Art center Ethiopia.
+          </p>
+        </li>
+        <li>
+          <div>
+            <img src="./images/forum.svg" alt="form logo " />
+          </div>
+          <span>Forum</span>
+          <p class="right">
+            Have the time to share your thoughts and opinions with experts for
+            each topic.
+          </p>
+        </li>
+        <li>
+          <div>
+            <img class="" src="./images/workshop.svg" alt="workshop logo " />
+          </div>
+          <span>workshop</span>
+          <p class="right">
+            Try creating your own work business rather than working for others.
+          </p>
+        </li>
+        <li>
+          <div>
+            <img class="" src="./images/ignite.svg" alt="Ignaite logo " />
+          </div>
+          <span>AWE Ignite</span>
+          <p class="right">
+            Get opportunities to network with AWE affiliates around the world,
+            also after the summit.
+          </p>
+        </li>
+      </ul>
+      <div class="link">
+        <a href="#" class="join-awe-mobile">Join AWE Global Summit 2023</a>
+        <a href="#" class="join-awe-desktop">Join AWE Global Summit 2023</a>
+      </div>
+    </section>
+
+    <section class="speakers">
+      <h3>Speakears</h3>
+      <hr />
+      <ul class="container"></ul>
+      <div class="more">MORE<i class="fa-solid fa-chevron-down"></i></div>
```

Comment:
- [ ] For semantic purposes, kindly use a `button` tag here instead of using a `div` which functions as a button.